modules/third-party/org-outerj-daisy-diff/patches/LPS-199999.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/org/outerj/daisy/diff/helper/CleanBrokenAttributeQNamesHandler.java b/org/outerj/daisy/diff/helper/CleanBrokenAttributeQNamesHandler.java
```

Comment:
@dantewang The file name LPS-199999.patch is wrong.

Reply:
@tinatian Ah sorry I forgot to change it. Will resend.